inference_rules.adoc

```diff
@@ -123,6 +123,7 @@ If a submission failed an audit that was delayed past publication, then any publ
 
 MLCommons shall retain a library of past audit reports and send copies to MLCommons members, auditors, and potential auditors by request. Audit reports will not be further distributed without permission from the audited submitter.
 
+An audit is expected to be completed within a 90 day period. Audits failing to meet this timeline will be nullified by request of the auditee.
```

Comment:
Once the auditor is chosen, the submitter has a great deal of control over the timeline - they're primarily responsible for constructing an NDA the auditor can sign, and primarily responsible for providing the information the auditor needs to produce the audit. So a submitter could run out the clock if they so choose.

My understanding of what was discussed in the meeting was that the submitter can request to the WG that the audit be nullified (and then presumably they'd bring good reasons why it didn't happen.)

Reply:
That makes sense. Updated to clarify that the submitter can request for a invalidation and the final decision rests with the Working Group.